host/basic_host.py

```diff
@@ -56,3 +69,22 @@ def set_stream_handler(self, protocol_id, stream_handler):
         """
         stream = await self.network.new_stream(peer_id, protocol_ids)
         return stream
+
+    async def connect(self, peer_info):
```

Comment:
Adding [reference](https://github.com/libp2p/go-libp2p/blob/master/p2p/host/basic/basic_host.go#L398) if anyone is wondering where this comes from.